draft-wussler-openpgp-pqc.md

```diff
@@ -365,21 +302,21 @@ The algorithms defined in this extension were chosen for standardization by the
 National Institute of Standards and Technology (NIST) in mid 2022
 {{NISTIR-8413}} as the result of the NIST Post-Quantum Cryptography
 Standardization process initiated in 2016 {{NIST-PQC}}. Namely, these are
-CRYSTALS-Kyber as a Key Encapsulation Mechanism (KEM), a KEM being a modern
-building block for public-key encryption, and CRYSTALS-Dilithium as well as
-SPHINCS+ as signature schemes.
+ML-KEM (formerly CRYSTALS-Kyber) as a Key Encapsulation Mechanism (KEM), a KEM
+being a modern building block for public-key encryption, and ML-DSA (formerly
+CRYSTALS-Dilithium) as well as SLH-DSA (formerly SPHINCS+) as signature
+schemes.
 
-For the two CRYSTALS-* schemes, this document follows the conservative strategy
-to deploy post-quantum in combination with traditional schemes such that the
+For the two ML-* schemes, this document follows the conservative strategy to
+deploy post-quantum in combination with traditional schemes such that the
 security is retained even if all schemes but one in the combination are broken.
-In contrast, the hashed-based signature scheme SPHINCS+ is considered to be
-sufficiently well understood with respect to its security assumptions in order
-to be used standalone. To this end, this document specifies the following new
-set: SPHINCS+ standalone and CRYSTALS-* as composite with ECC-based KEM and
+In contrast, the stateless hash-based signature scheme SLH-DSA is considered to
+be sufficiently well understood with respect to its security assumptions in
+order to be used standalone. To this end, this document specifies the following
+new set: SLH-DSA standalone and ML-* as composite with ECC-based KEM and
```

Comment:
```suggestion
new set: SLH-DSA standalone and two ML-* as composite with ECC-based KEM and
```

Using regular expression like syntax in normal text is not necessarily common or intuitive to readers. Adding the word "two" makes it at least clearer that this is refering to two schemes and not denoting a specific one.